### PR TITLE
Add DSARate bulk upload

### PR DESCRIPTION
**Pull request by ilcsik**
Connects https://github.com/unicef/etools-issues/issues/58

### CODE REVIEW

**Comment by ilcsik** [comment]
Considered ready for review but needs to be tested with initial data loaded (countries etc)

**Comment by ilcsik** [comment]
To be tested with newest csv

File: EquiTrack/publics/admin.py [hunk 1]
```diff
@@ -1,6 +1,12 @@
 from __future__ import unicode_literals
 
+import json
+
 from django.contrib import admin
+from django.utils.safestring import mark_safe
+from pygments import highlight
```
Comment (ntrncic) [inline EquiTrack/publics/admin.py]: Do we really pygments? We don't have to return pretty errors as this is done in admin.

Comment (ilcsik) [inline EquiTrack/publics/admin.py]: Just wanted to make a bit more friendly as these errors are stored in JSON and it's quite unreadable in plain text (basically no indent and stuff)

**Comment by robertavram** [comment]
@simon-the-sloth please review this PR

**Comment by ilcsik** [comment]
@simon-the-sloth sure I'll figure out!

**Pull request merged**